src/material/datepicker/calendar-body.ts

```diff
@@ -142,6 +142,10 @@ export class MatCalendarBody implements OnChanges, OnDestroy, AfterViewChecked {
 
   @Output() readonly activeDateChange = new EventEmitter<MatCalendarUserEvent<number>>();
 
+  @Output() readonly dragStarted = new EventEmitter<MatCalendarUserEvent<any /* D */>>();
```

Comment:
I would be nice if we could type this better. Is it difficult to add a generic to this class?